blocks/block-edit/index.js

```diff
@@ -17,7 +21,7 @@ function BlockEdit( props ) {
 	// them preferencially as the render value for the block.
 	const Edit = blockType.edit || blockType.save;
 
-	return applyFilters( 'BlockEdit', <Edit key="edit" { ...editProps } />, props );
+	return <Edit { ...editProps } />;
 }
 
-export default BlockEdit;
+export default withFilters( 'BlockEdit' )( BlockEdit );
```

Comment:
since the filters are "global", we should think of namespacing them: `core/blocks/BlockEdit`. It's not allowed right now in the hooks package and it's not required for this PR I think.

Reply:
We can put `Blocks.BlockEdit` until `hooks` package gets updated.